RetroAchievements/RAWeb
# chore: make Inertia SSR port configurable via .env

Still uses 13714 as the default, but can now be set via `VITE_INERTIA_SSR_PORT`.

This change should theoretically allow us to resolve an issue where the Inertia SSR daemon is using the same port in all 3 of our server's active environments.

**How to test this PR:**
In your _.env_, add the following line:
```
VITE_INERTIA_SSR_PORT=13715
```
Then:
```
sail npm run build
sail artisan inertia:start-ssr
```

The Inertia instance should start in your terminal and report that it's running on port 13715. You should be able to navigate to `/game/1/hashes` and not see any console errors.

---

I have the Inertia daemons currently disabled in both preview and stage. After this PR is merged, we can probably try to re-enable them again and see how it goes. **We may need to be prepared to restart nginx again when this happens if something goes awry.**
